Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1431,7 +1431,8 @@ final class StatusTableViewController: LoopChartsTableViewController {
         let hostingController = DismissibleHostingController(
             content: bolusEntryView(
                 enableManualGlucoseEntry: enableManualGlucoseEntry
-            )
+            ),
+            guidanceColors: .default
```

Comment:
I see that elsewhere a `DismissibleHostingController` is instantiated using 

```
DismissibleHostingController(rootView: ..., isModalInPresentation: ...)
```

which uses the .default guidance colors. Should this be used here as well?

Reply:
Good question! Let me take a look! This was the only non-preview instance of DismissableHostingController that didn't have the proper color palette.

Reply:
Oh yeah, that's much better.